lib/build/build.go

```diff
@@ -36,6 +36,9 @@ var (
 	// Set by Go build tags
 	Tags []string
 
+	// Set by the caller
+	program = "syncthing"
```

Comment:
I guess using `Syncthing` for the proper "brand name" would be too scary (as there might be places where `syncthing` is expected/hardcoded).

Reply:
Yeah... Breaking change, reserved for 2.0!